README.md

```diff
@@ -361,7 +361,7 @@ kustomize build apps/xgboost-job/upstream/overlays/kubeflow | kubectl apply -f -
 
 #### User Namespace
 
-Finally, create a new namespace for the the default user (named `user`).
+Finally, create a new namespace for the the default user (named `user-example-com`).
```

Comment:
The namespace name is `kubeflow-user-example-com`, not `user-example-com`. How about we include both the user name and the namespace name?

Old:

Finally, create a new namespace for the the default user (named `user-example-com`).

New: 

Finally, create a new namespace (named `kubeflow-user-example-com`) for the the default user (named `user@example.com`).